s2i/builder.go

```diff
@@ -386,5 +382,18 @@ func builderImage(f fn.Function) (string, error) {
 		return v, nil
 	}
 
-	return "", ErrRuntimeNotSupported
+	return "", ErrRuntimeNotSupported{f.Runtime}
+}
+
+func IsErrRuntimeNotSupported(err error) bool {
+	var e ErrRuntimeNotSupported
+	return errors.As(err, &e)
+}
+
+type ErrRuntimeNotSupported struct {
+	Runtime string
+}
+
+func (e ErrRuntimeNotSupported) Error() string {
+	return fmt.Sprintf("the s2i builder has no default builder image for the %q language runtime", e.Runtime)
```

Comment:
Do you think it's useful to indicate here that buildpacks can be used if the runtime is one that is supported OOTB? Otherwise, lgtm

Reply:
Changed. @lance PTAL